about.html

```diff
@@ -0,0 +1,154 @@
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer">
+    <link rel="stylesheet" href="about.css">
+    <link rel="stylesheet" href="style.css">
+    <title>About Section</title>
+</head>
+<body>
+    <!-- -------------------------Header section Starts------------------------------->
+    <section class="header-one">
+        <ul class="padding">
+            <li><i class="fab fa-facebook-f"></i></li>
+            <li><i class="fab fa-twitter"></i></li>
+            <li>english</li>
+            <li>my page</li>
+            <li>logout</li>
+        </ul>
+    </section>
+    <header class="header-two">
+        <div class="logo">
+            <ul>
+                <li>
+                    <a href="index.html"><img src="images/logo.png" alt=""></a>
+                </li>
+            </ul>
+            <span>MilCon 2023</span>
+        </div>
+        <div id="icon" class="icon fas fa-bars"></div>
```

Comment:
- Please, this content should be wrapped in the nav tag. I've left the reason why you should use the nav tag. kindly check in my previous review above.

Reply:
@zieeco Thanks for the suggestion. Please check the commit message. It is fixed in about.html file from lines 22 - 29.